Gemfile.lock

```diff
@@ -36,4 +37,4 @@ DEPENDENCIES
   rspec
 
 BUNDLED WITH
-   1.14.4
+   1.14.6
```

Comment:
What happened here? Not a big concern, but wondering why your Gemfile.lock updated. Might point to an issue with your setup. Don't do this when you're on a project with a team! Everyone will already have their gems all set and working with an agreed upon version.